modules/lib/font-type.nix

```diff
@@ -0,0 +1,28 @@
+{ lib }:
+
+with lib;
+
+{
+  fontType = types.submodule {
```

Comment:
Can move this to `types.nix` instead.

Reply:
Should I be able to call it directly through `lib.hm.types` once moved to `types.nix`? I've only managed to call it like done [here](https://github.com/rycee/home-manager/blob/master/modules/programs/emacs.nix#L39).

Reply:
Yes, `lib.hm.types.fontType` should work. Like here: https://github.com/rycee/home-manager/blob/95c8007b8fe4433cd217db83ce26b8e29875ba06/modules/home-environment.nix#L235

Reply:
It works, I think my fork wasn't up to date when I last tried.